lib/rubocop/cop/style/stderr_puts.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop identifies places where `$stderr.puts`
+      # can be replaced by `warn`.
+      #
+      # @example
+      #   # bad
+      #   $stderr.puts('hello')
+      #
+      #   # good
+      #   warn('hello')
+      #
+      class StderrPuts < Cop
+        MSG = 'Use `$stderr.puts` instead of `warn`.'.freeze
```

Comment:
Should we use `warn` instead of `$stderr.puts`? It looks reversed.

Reply:
Oops! Thanks for review.